src/routes/(console)/organization-[organization]/usage/[[invoice]]/+page.svelte

```diff
@@ -143,9 +143,11 @@
         <Heading tag="h6" size="7">Users</Heading>
 
         <p class="text">The total number of users across all projects in your organization.</p>
-
         <svelte:fragment slot="aside">
             {#if data.organizationUsage.users}
+                {@const users = data.organizationUsage.users.filter(
+                    (user) => new Date(user.date) < new Date()
+                )}
```

Comment:
Would it make more sense to move this on the `.ts` side? I see bunch of calc already being handled there. wdyt?

Reply:
Yeah would also make more sense typing wise. I'll look into it!